tests/test_routes.py

```diff
@@ -216,6 +224,24 @@ def test_route_fund_dashboard(
         ]
         assert all(label in all_filter_labels for label in expected_filter_labels)
 
+        # Check the 'assigned to you' tab
+        assigned_to_you_table = (
+            soup.find("div", id="assigned-to-you")
+            .find("table", id="application_overviews_table")
+            .find("tbody")
+        )
+
+        expected_values = [
```

Comment:
could probs add the user data in here i think?

Reply:
Ah yes, missed that!